db/migrate/20160514212340_change_suspended_on_homes_to_enum.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeSuspendedOnHomesToEnum < ActiveRecord::Migration
+  def change
+    remove_column :homes, :suspended
+    add_column :homes, :status, :integer, default: 0
+  end
```

Comment:
Yeah I should have commented on the diff about the change. I did that because we had the methods